cmd/network/verification_test.go

```diff
@@ -64,20 +64,56 @@ func Test_egressVerificationSetup(t *testing.T) {
 		expectErr bool
 	}{
 		{
-			name: "no ClusterId requires subnet/sg",
+			name:      "no ClusterId requires subnet/sg",
+			e:         &EgressVerification{},
+			expectErr: true,
+		},
+		{
+			name:      "no ClusterId with subnet",
+			e:         &EgressVerification{SubnetIds: []string{"subnet-a"}},
+			expectErr: true,
+		},
+		{
+			name:      "no ClusterId with security group",
+			e:         &EgressVerification{SecurityGroupId: "sg-a"},
+			expectErr: true,
+		},
+		{
+			name:      "no ClusterId with platform type",
+			e:         &EgressVerification{PlatformType: "aws"},
+			expectErr: true,
+		},
+		{
+			name: "no ClusterId with subnet and security group",
 			e: &EgressVerification{
-				ClusterId: "",
+				SubnetIds:       []string{"subnet-a", "subnet-b", "subnet-c"},
+				SecurityGroupId: "sg-b",
+			},
+			expectErr: true,
+		},
+		{
+			name: "no ClusterId with subnet and platform type",
+			e: &EgressVerification{
+				SubnetIds:    []string{"subnet-a", "subnet-b", "subnet-c"},
+				PlatformType: "aws",
+			},
+			expectErr: true,
+		},
+		{
+			name: "no ClusterId with security group and platform type",
+			e: &EgressVerification{
+				SecurityGroupId: "sg-b",
+				PlatformType:    "aws",
 			},
 			expectErr: true,
 		},
 		{
 			name: "ClusterId optional",
 			e: &EgressVerification{
-				ClusterId:       "",
 				SubnetIds:       []string{"subnet-a", "subnet-b", "subnet-c"},
 				SecurityGroupId: "sg-b",
+				PlatformType:    "aws",
 			},
-			expectErr: false,
```

Comment:
I think we should put this back. Yes, the behavior is the same with or without the line, but it's more obvious what the intent of the test is if `expectErr: false` is here.